index.html

```diff
@@ -165,97 +165,136 @@ <h3 id="heading">Test Run for Firefox RHEL6.5 (Desktop)</h3>
-
-                    shown.className = 'img hidden-right';
-                    hidden.className = 'img shown';
-
-                    hidden.style.backgroundImage = 'url(' + imgs[imgIndex].img + ')';
-                    heading.textContent = imgs[imgIndex].title;
-                    subtitle.textContent = imgs[imgIndex].subtitle;
-
-                    shown = document.getElementsByClassName('shown')[0];
-                    hidden =  document.getElementsByClassName('hidden-right')[0];
-                });
-
-                prevBtn.addEventListener('click', function(){
-                    imgIndex--;
-
-                    if(imgIndex < 0){
-                        imgIndex = imgsLength - 1;
-                    }
-
-                    index.textContent = (imgIndex + 1) + '/' + imgsLength;
-
-                    shown.className = 'img hidden-left';
-                    hidden.className = 'img shown';
-
-                    hidden.style.backgroundImage = 'url(' + imgs[imgIndex].img + ')';
-                    heading.textContent = imgs[imgIndex].title;
-                    subtitle.textContent = imgs[imgIndex].subtitle;
-
-                    shown = document.getElementsByClassName('shown')[0];
-                    hidden =  document.getElementsByClassName('hidden-left')[0];
-                });
+        //Imgs array and other variables needed for the slider logic :)
+        var imgs = [
+            {
+                title: 'Create IEEE 829 or free text Test Plan',
+                subtitle: 'This document describes what and how are you going to test',
+                img: 'images/screenshots/001_create_test_plan.png'
+            }, {
+                title: 'Create a Test Case',
+                subtitle: 'Use Given-When-Then or bullet points to describe the scenario',
+                img: 'images/screenshots/002_create_test_case.png'
+            }, {
+                title: 'Execute Test Runs against a build',
+                subtitle: 'for each build create test runs to collect execution details',
+                img: 'images/screenshots/003_execute_tests_for_build.png'
+            }, {
+                title: 'Mark individual test result statuses',
+                subtitle: 'add comments, link or report bugs',
+                img: 'images/screenshots/004_mark_individual_test_results.png'
+            }, {
+                title: 'Product builds report',
+                subtitle: 'shows testing progress across builds',
+                img: 'images/screenshots/005_product_builds_report.png'
+            }, {
+                title: 'Testing report per Test Plan and product build ',
+                subtitle: 'shows detailed statuses per build and test run',
+                img: 'images/screenshots/006_report_per_plan_and_build.png'
+            }],
+            imgIndex = 0,
+            imgsLength = imgs.length;
+				
+        //Slider control elements	
```

Comment:
You still have some leading and trailing TABs between lines. Make sure to remove them, use empty new line to separate code.

Reply:
Ops! Sorry i missed these